src/aave-v2/MorphoGovernance.sol

```diff
@@ -467,7 +467,7 @@ abstract contract MorphoGovernance is MorphoUtils {
         Types.MarketPauseStatus storage pause = marketPauseStatus[_poolToken];
 
         pause.isSupplyPaused = _isPaused;
-        pause.isBorrowPaused = _isPaused;
+        if (!pause.isDeprecated) pause.isBorrowPaused = _isPaused;
```

Comment:
should we add the note // Note that pause.isDeprecated implies pause.isBorrowPaused.